### ISSUE TITLE
Ways to build measures

### ISSUE THREAD

**Issue by cscherrer** [cscherrer/MeasureTheory.jl#59]
- [ ] From a log-density wrt some base measure
- [ ] As a superposition of other measures
- [ ] As a product measure
- [ ] By normalizing an existing measure
- [ ] From a CDF or copula function
- [ ] "Importance sampling" representation - changing the base measure

**Comment by sethaxen** [comment]
There's also using expectations of basis functions. E.g. characteristic functions. Useful for defining a measure as a convolution of measures.

**Comment by cscherrer** [comment]
Thanks @sethaxen , I've added it

**Comment by sethaxen** [comment]
Also one might want to define a measure from some function that computes random samples.

**Comment by cscherrer** [comment]
Nice @sethaxen . This one will probably be tricky, maybe @zenna's work could help with it?

**Comment by sethaxen** [comment]
> This one will probably be tricky, maybe @zenna's work could help with it?

Yeah, I'm just listing things a user might want to do, not necessarily that will be easy to support. I'm not familiar with @zenna's work.

Sam Power tweeted out this chapter a few days ago, and it had some additional ways of generating random variables that I hadn't seen before: http://luc.devroye.org/handbooksimulation1.pdf

**Comment by cscherrer** [comment]
Nice, reminds me of this:
https://blog.shakirm.com/2015/10/machine-learning-trick-of-the-day-4-reparameterisation-tricks/
